lms/models/grading_sync.py

```diff
@@ -30,6 +30,8 @@ class GradingSync(CreatedUpdatedMixin, Base):
 
     status: Mapped[str | None] = varchar_enum(AutoGradingSyncStatus)
 
+    grades: Mapped[list["GradingSyncGrade"]] = relationship()
```

Comment:
We use this in the code base but it's defined via a `backref`. Adding the explicit relationship here.